src/handlers/channel/readyPermissions.js

```diff
@@ -0,0 +1,53 @@
+import 'dotenv/config';
+
+const readyPermissions = async (client) => {
+  console.log(`Logged in as ${client.user.tag}!`);
+  try {
+    // '스스로 만들기' 카테고리의 ID를 확인합니다.
+    const guild = client.guilds.cache.get(process.env.SERVER_ID); // 해당 서버의 ID를 넣어야 합니다.
+    const category = guild.channels.cache.find((ch) => ch.name === '스스로 만들기');
```

Comment:
나중에 권한을 조정하고싶은 카테고리가 여러개일 수도 있으니까 선택된 카테고리를 상수배열로 분리시키고 로직을 `includes`를 활용하여 바꿀 수 있을 것 같아요
```suggestion
    const category = guild.channels.cache.find((ch) => SELECTED_CATEGORY.includes(ch.name);
```

Reply:
좋습니다~!
```js
client.on('ready', async () => {
  const SELECTED_CATEGORY = '원하는 카테고리 명'
  await readyPermissions(client, SELECTED_CATEGORY);
});
```